exchanges/account/account_types.go

```diff
@@ -76,3 +82,9 @@ type ProtectedBalance struct {
 	// usage.
 	notice alert.Notice
 }
+
+// Protected limits the access to the underlying credentials outside of this
+// package and allows it to be nil.
```

Comment:
`Protected` is never used as a pointer and the `creds` field is not a pointer, so it cannot be nil? Maybe I'm missing something like my first review 😄

Reply:
No, you didn't, it used to be a pointer captured in a for loop, and there were some unforeseen consequences. Will remove.

Reply:
No, you didn't, it used to be a pointer captured in a for loop, and there were some unforeseen consequences. Will remove.